src/app-logic/constants.js

```diff
@@ -9,7 +9,7 @@ import type { MarkerPhase } from 'firefox-profiler/types';
 // The current version of the Gecko profile format.
 // Please don't forget to update the gecko profile format changelog in
 // `docs-developer/CHANGELOG-formats.md`.
-export const GECKO_PROFILE_VERSION = 27;
```

Comment:
Yeah, it could be good indeed. Before this changelog file it was our only source of truth, and it should still be one, especially for more implementation details etc. (as oppose to changelog)

Reply:
Good idea @julienw. I'll add one and update the patch.

Reply:
Good idea @julienw. I'll add one and update the patch.